tests/src/Simple/HelloWasm/HelloWasm.csproj

```diff
@@ -7,6 +7,7 @@
 	
 	<ProjectReference Include="ILHelpers.ilproj" Condition="'$(OS)' == 'Windows_NT'" />
     <IlcArg Include="-r:$(IntermediateOutputPath)\ILHelpers.dll" Condition="'$(OS)' == 'Windows_NT'" />
+    <ILcArg Include="--nometadatablocking" />
```

Comment:
Why is this necessary? Metadata blocking is about reflection-blocking the implementation details of the framework that nobody should be reflecting on. It saves disk space to make those things invisible to reflection. Especially for a target like WASM, saving space is good.

Reply:
Its not, I incorrectly thought I needed that to get a policy that created meta data but as you say its not needed.

Reply:
Could you also remove the references to the switch in the docs this pull request would be adding?

Reply:
Thanks, forgot that.